src/View/View.php

```diff
@@ -651,7 +653,8 @@ public function beginCache(string $id, array $properties = []): bool
     {
         $properties['id'] = $id;
         $properties['view'] = $this;
-        $cache = FragmentCache::begin($properties);
+        $cache = FragmentCache::begin();
```

Comment:
Doesn't seem to be related.

Reply:
The widgets do not support configuration of array, now we use functions in stack.